saleor/order/migrations/tasks/saleor3_18.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import F, Func, OuterRef, Subquery
+
+from ....celeryconf import app
+from ...models import Order, OrderLine
+
+# celery task for one batch took 0.1s and 10.54MB at its peak
+BATCH_SIZE = 1000
+
+
+@app.task
+def update_order_subtotals(batch_id=0):
+    start_order_number = batch_id * BATCH_SIZE
+    end_order_number = start_order_number + BATCH_SIZE
```

Comment:
What if the order numbers start from 2000? In such case, we won't update any orders

Reply:
I applied changes to fix that edge case, it is in here https://github.com/saleor/saleor/pull/14932/commits/96ec0b393064d16791ff1bc5f0bf8e5345520c1b